src/styles/default.css

```diff
@@ -92,7 +92,7 @@ main {
 }
 
 main p {
-  width: clamp(400px, 80%, 700px);
```

Comment:
This is creating some funky styling on mobile. I don't think the low bound is low enough? Changing to `max-width` seems to make things flow responsively well.